app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -12,6 +12,10 @@
   <meta name="govuk:navigation-page-type" content="Taxon Page" />
 <% end %>
 
+<script type="application/ld+json">
+  <%= raw special_announcement.payload.to_json %>
```

Comment:
Shall we put this in `app/views/coronavirus_landing_page/_schema.html.erb`?

Reply:
Yep, good shout - I'd just rebased and hadn't got round to aligning with you yet :-)